pkg/event/target/mqtt.go

```diff
@@ -140,8 +156,25 @@ func NewMQTTTarget(id string, args MQTTArgs) (*MQTTTarget, error) {
 
 	client := mqtt.NewClient(options)
 	token := client.Connect()
-	if token.Wait() && token.Error() != nil {
-		return nil, token.Error()
+
+	if args.QueueDir != "" {
+		go func() {
+			// Repeat the pings until the client registers the token.
+			for {
+				if token.Wait() && token.Error() == nil {
+					fmt.Println("MQTT: Connected")
```

Comment:
remove all debug prints.

Reply:
This was intentional to let the user know that we are retrying. Should i remove this ?

Reply:
Done